components/Blog/Card/Img.tsx

```diff
@@ -0,0 +1,9 @@
+import Image, { ImageProps } from 'next/image';
+
+interface Props extends ImageProps { }
+
+const Img = ({ ...props }: Props) => {
```

Comment:
We dont need to divide BlogCard into seperate components. You can use Compound Component Pattern for createing it more simple way. Create only BlogCard file and implement all componnents into this file. And delete all other files.You can learn it by watching this video or you can inspect codes on this repository. https://youtu.be/4xFf_KiBE6A